stable/rails/values.yaml

```diff
@@ -25,12 +25,20 @@ migrations:
   - "bundle exec rake db:migrate"
 
 ingress:
-  enabled: false
-  host: myhost.local
-  annotations: {}
-  tls:
-    enabled: false
-    secret: myhost.local-tls
+  enabled: true
```

Comment:
I don't like this `enabled` and then `list`. Maybe the following config could be easier:
```
ingress:
  - host:
  - host:
```
However at the same time not insisting to change it. Up to you